### ISSUE TITLE
Using config file prevents `Clamby.update`

### ISSUE THREAD

**Issue by PrinceUdoka** [kobaltz/clamby#46]
I run clamd as a docker container and
when I defined the `:config_file` in the Clamby initialiser I used the `clamd` config.
Then it errors out when trying to run `Clamby.update`, 
since the `clamd` config has options that `freshclam` does not.

So it would be preem if there was a way to use a second config for freshclam or something else. 🙂

**Comment by henryaj** [comment]
Yep, this seems essential as I'm finding they can't share a config.

**Comment by henryaj** [comment]
I've updated this in my fork - https://github.com/henryaj/clamby
